.travis/yaml_splitter.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python
```

Comment:
From google style on shell scripts (https://google.github.io/styleguide/shell.xml#Shell_Files_and_Interpreter_Invocation):

>  It is not necessary to know what language a program is written in when executing it and shell doesn't require an extension so we prefer not to use one for executables.

I think that would mean `yaml_splitter.py` would be renamed to `yaml_splitter`, the sha-bang line would be updated to `#!/usr/bin/python3`, and the invocation updated to `./.travis/yaml_splitter`